ec2.tf

```diff
@@ -25,6 +25,10 @@ sudo yum -y update
 EOF
 }
 
+locals {
+  tags = var.bastion_tags
+}
+
```

Comment:
You don't need the locals here, just set the `for-each` equal to the variable you created. Comment below.